assets/locales/service.en.toml

```diff
@@ -298,7 +298,11 @@ one = "These statistics have been given National Statistics status by the Office
 
 [AboutDataNationalStatisticsBody2]
 description = "They meet the highest standards of trustworthiness, quality and public value. This means that they:"
-one = " They meet the highest standards of trustworthiness, quality and public value. This means that they:"
+one = " They meet the highest standards of trustworthiness, quality and public value."
```

Comment:
Should we amend de description and remove the `This means that they:` bit

Reply:
this is done.